client/battle/CBattleInterface.cpp

```diff
@@ -404,7 +406,7 @@ CBattleInterface::CBattleInterface(const CCreatureSet *army1, const CCreatureSet
 	battleIntroSoundChannel = CCS->soundh->playSoundFromSet(CCS->soundh->battleIntroSounds);
 	auto onIntroPlayed = [&]()
 	{
-		if(LOCPLINT->battleInt)
+		if (LOCPLINT->battleInt)

```

Comment:
As far as I remember we do not use spaces after if/for and so on. Or am I wrong?

Reply:
fixed here